### PR TITLE
[xcode11.4] Initial commit for Xcode 11.4 Beta 3 support

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: tests/xtro-sharpie/macOS-NetworkExtension.todo [hunk 1]
```diff
@@ -13,11 +10,15 @@
...
-!missing-selector! NETunnelProviderManager::initForPerAppVPN not bound
 !missing-selector! NETunnelProviderManager::mailDomains not bound
 !missing-selector! NETunnelProviderManager::safariDomains not bound
 !missing-selector! NETunnelProviderManager::setAppRules: not bound
 !missing-selector! NETunnelProviderManager::setCalendarDomains: not bound
 !missing-selector! NETunnelProviderManager::setContactsDomains: not bound
 !missing-selector! NETunnelProviderManager::setMailDomains: not bound
 !missing-selector! NETunnelProviderManager::setSafariDomains: not bound
+!missing-enum! NEFilterReportFrequency not bound
+!missing-selector! +NETunnelProviderManager::forPerAppVPN not bound
```
Comment (whitneyschmidt) [inline tests/xtro-sharpie/macOS-NetworkExtension.todo]: This looks weird - `+NETunnelProviderManager`?

Comment (dalexsoto) [reply tests/xtro-sharpie/macOS-NetworkExtension.todo]: @whitneyschmidt why?

Comment (spouliot) [reply tests/xtro-sharpie/macOS-NetworkExtension.todo]: weird because it starts with a `+` ?
That means it's a static selector

Comment (whitneyschmidt) [reply tests/xtro-sharpie/macOS-NetworkExtension.todo]: 🤦‍♀ @spouliot thank you for the kind reminder...

**Pull request merged**